packages/gallery/src/components/item/imageRenderer.js

```diff
@@ -1,28 +1,28 @@
 import React from 'react';
 import { PrintOnlyImageSource } from './printOnlySource';
 
-const ImageRenderer = (props) => {
-  if (typeof ImageRenderer.customImageRenderer === 'function') {
-    return ImageRenderer.customImageRenderer(props);
-  } else if (typeof props.src === 'string') {
-    return <img alt={props.alt} {...props} />;
-  } else if (typeof props.src === 'object') {
+const ImageRenderer = ({ customImageRenderer, ...restProps }) => {
```

Comment:
why cant the customImageRenderer be extracted from props w/o this?
I dont like the fact that there are so many semantic changes to this file.